addons/udes_priorities/models/priority.py

```diff
@@ -26,6 +26,14 @@ class UdesPriority(models.Model):
         default=lambda self: uuid4(),
     )
 
+    @api.multi
+    @api.depends("reference")
+    def _get_default_affected_pickings(self):
+        Picking = self.env["stock.picking"]
+        self.affected_pickings = Picking.search(
+            [("priority", "=", self.reference), ("state", "not in", ("cancel", "done")), ]
```

Comment:
Nit: Extra space before the closing square bracket, has the file been formatted with black?

Reply:
No, I have switched to a different IDE (PyCharm) and haven't set up black formatting yet. Will definitely spend some time making it the default formatting of my IDE to avoid this.